sys/auto_init/auto_init.c

```diff
@@ -304,4 +309,12 @@ void auto_init(void)
     DEBUG("Auto init UDP module.\n");
     ng_udp_init();
 #endif
+#ifdef MODULE_NG_NETIF
+    DEBUG("Auto init netif module.\n");
+    ng_netif_init();
```

Comment:
This function does not need to be called. I think I have to put this into `TEST_SUITES` context :D